fusee/program/update_mtc_tables.py

```
@@ -57,6 +57,22 @@ def main(argc, argv):
             except:
                 pass
             write_file(os.path.join('mtc_tables/lz/%s' % board, '%d.lz4' % i), compressed)
+        try:
+            os.makedirs('mtc_tables/combined/%s' % board)
+        except:
+            pass
```

Ruff (0.8.2)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)